### ISSUE TITLE
AMD GPU support via PlaidML?

### ISSUE THREAD

**Issue by palacinka** [OlafenwaMoses/ImageAI#166]
would it be possible to use this library with PlaidML to support more GPUs? https://github.com/plaidml/plaidml

I tried to add
```
os.environ["KERAS_BACKEND"] = "plaidml.keras.backend"
```

but getting 
```
imageai\Detection\keras_retinanet\backend\dynamic.py", line 12, in <module>
    assert _backend in backends
AssertionError
```

**Comment by OlafenwaMoses** [comment]
This will be considered and may be added in future versions.

**Comment by nmatton** [comment]
Also interested ! Is there any workaround we can set up in the mean time  @OlafenwaMoses ?
Thanks !

**Comment by whackamadoodle3000** [comment]
I'm interested too... Without an Nvidia GPU I'd have to pay for GCP or something similar but it would be nice to take advantage of my existing AMD one. Thanks!
